packages/automerge-repo/src/Repo.ts

```diff
@@ -158,8 +158,10 @@ export class Repo extends EventEmitter<RepoEvents> {
       const saveFn = ({ handle, doc }: DocHandleEncodedChangePayload<any>) => {
         void this.storageSubsystem!.saveDoc(handle.documentId, doc)
       }
+      const keyFn = ({ handle }: DocHandleEncodedChangePayload<any>) => handle
```

Comment:
This assumes that the `handle` object is stable per document ID. If that's not true, we can modify `throttleByKey` to use a regular map rather than a `WeakMap`, and use the document ID as a key.

Reply:
Looking at how handles are managed in this class, I think that's a safe assumption.